lib/tailwind_formatter.ex

```diff
@@ -43,37 +43,17 @@ defmodule TailwindFormatter do
       |> Inspect.Algebra.format(:infinity)
       |> IO.iodata_to_binary()
 
-    sorted_classes =
-      if String.contains?(sorted_classes, "\""),
-        do: trim_classes(sorted_classes),
-        else: sorted_classes
-
     String.replace(contents, expr_class, sorted_classes)
   end
 
-  defp trim_classes(classes) do
-    classes
-    |> String.trim("\"")
-    |> String.trim(" ")
-    |> then(fn str -> "\"#{str}\"" end)
-  end
-
   defp sort_expr({:<>, meta, children}), do: {:<>, meta, handle_concatenation(children)}
   defp sort_expr({:<<>>, meta, children}), do: {:<<>>, meta, handle_interpolation(children)}
   defp sort_expr({:__block__, meta, [children]}), do: {:__block__, meta, [sort_expr(children)]}
   defp sort_expr(text) when is_binary(text), do: sort(text)
   defp sort_expr(node), do: node
 
   defp handle_concatenation(children) do
-    children
-    |> Enum.map(&sort_expr/1)
-    |> Enum.map(fn
-      {:__block__, meta, [text]} when is_binary(text) ->
-        {:__block__, meta, [" #{text} "]}
```

Comment:
My first attempt at handling leading and trailing space was here. I replaced this with something that adds a leading or trailing space depending on whether the child is on the left or right of `<>` and on whether the child was a block of text or not. That initially worked but then fell apart when concatenations were next to interpolations.